audio-worklet/src/bitcrusher-demo.js

```diff
@@ -124,13 +150,13 @@ class BitcrusherDemo {
    */
   switchBitcrusherNode() {
     // The function is called after the checkbox switches state.
-    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
-    if (changeToScriptProcessor) {
-      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
-      this.bitcrusherAudioWorklet_.disconnect();
-    } else {
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
```

Comment:
I answered it the previous time you brought it up. It's because the event listener calls the function after it has switched boolean value.